scripts/Dockerfile

```diff
@@ -15,6 +15,9 @@
 ARG DEPENDENCIES_IMAGE="sleeper-dependencies:current"
 FROM ${DEPENDENCIES_IMAGE}
 
+RUN curl -SL https://github.com/docker/compose/releases/latest/download/docker-compose-linux-x86_64 \
+    -o /usr/local/bin/docker-compose
+RUN chmod +x /usr/local/bin/docker-compose
```

Comment:
I think Docker Compose is already installed, you probably need to use `docker compose` instead of `docker-compose`.